lib/super_spreader/stop_signal.rb

```diff
@@ -3,15 +3,21 @@
 require "redis"
 
 module SuperSpreader
+  # @deprecated Please use TrackBallast::StopSignal instead
   module StopSignal
+    # @deprecated Please use {TrackBallast::StopSignal.stop!} instead
     def stop!
+      warn "DEPRECATION WARNING: the class SuperSpreader::StopSignal is deprecated and will be removed in v1.0. " \
+        "Use TrackBallast::StopSignal instead."
```

Comment:
There's also the [built-in `deprecate` method](https://ruby-doc.org/core-3.1.2/Gem/Deprecate.html), which may provide some advantages.  I'm unsure, to be honest.  Do you have an opinion on this option?  (I could go either way.)